src/type.c

```diff
@@ -143,17 +155,15 @@ SEXP vctrs_type_common(SEXP call, SEXP op, SEXP args, SEXP env) {
 
 SEXP vctrs_type_common_impl(SEXP dots, SEXP ptype) {
   if (!vec_is_partial(ptype)) {
-    return vec_type(ptype);
+    return vec_ptype(ptype, args_dot_ptype);
   }
 
   if (r_is_true(r_peek_option("vctrs.no_guessing"))) {
     Rf_errorcall(R_NilValue, "strict mode is activated; you must supply complete `.ptype`.");
   }
 
   // Start reduction with the `.ptype` argument
-  struct vctrs_arg ptype_arg = new_wrapper_arg(NULL, ".ptype");
-
-  SEXP type = PROTECT(reduce(ptype, &ptype_arg, dots, &vctrs_type2_common));
+  SEXP type = PROTECT(reduce(ptype, args_dot_ptype, dots, &vctrs_type2_common));
```

Comment:
(That's a case where the `p_` convention would be helpful, e.g. `p_args_dot_ptype`.)

Reply:
Yea normally I would, but I went for matching `args_empty` here

Reply:
This was more of a comment about the `args_` prefix generally.